### PR TITLE
Ship 2024 site

### PR DESCRIPTION
**Pull request by ericholscher**
This is a WIP preview to get PR previews

----
:books: Documentation preview :books:: https://writethedocs-www--2051.org.readthedocs.build/conf/portland/2024/

### CODE REVIEW

**Comment by plaindocs** [comment]
Looking at this PR, can we replace the "has sponsors" flag with check for content in the sponsors file? Or not worth the trouble CC @mxsasha

File: docs/_data/portland-2024-config.yaml [hunk 1]
```diff
@@ -107,23 +107,23 @@ date: # how do we handle these? human readable would be nice.
 
   # Conference is special for the index to show the combined conference days
   conference:
-    event: Main Conference
+    event: Speaker talks
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml:110]: This does not roll of the tongue. How about?

```suggestion
    event: Scheduled talks
```

File: docs/_data/portland-2024-config.yaml [hunk 2]
```diff
@@ -107,23 +107,23 @@ date: # how do we handle these? human readable would be nice.
...
     date: April 15-16
-    summary: The main days of the conference. We will be running one main track of talks, as well as a <a href="/conf/portland/2024/job-fair/">Sponsor Expo</a>, <a href="/conf/portland/2024/lightning-talks/">Lightning Talks</a> & <a href="/conf/portland/2024/unconference/">Unconference</a>.
+    summary: The main days of the conference. Listen to a diverse panel of speakers share their insights and experience. Currently accepting Call for Proposals.
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml]:
```suggestion
    summary: The main days of the conference. Our speakers share their insights and experience, Other events include unconference discussions, lighting talks and job fair. 
```

File: docs/_templates/2024/index.html [hunk 3]
```diff
@@ -1,11 +1,12 @@
 {% extends year_str + "/base.html" %}
 
 {% block full_title %}
-Home - Write the Docs {{ name }} {%- if cobrand is defined and cobrand %} + {{ cobrand.brand }} {% endif %} {{ year }}
+Home - Write the Docs {{ name }} {{ year }}
```
Comment (plaindocs) [inline docs/_templates/2024/index.html:4]: :+1:

File: docs/_templates/2024/index.html [hunk 4]
```diff
@@ -183,13 +181,47 @@ <h3>{{ date.conference.event }} </h3>
...
+    {% endif %}
+
+      {# {% if flagspeakersannounced %} #}
 
       <div class="learn-more">
         <a class="uk-align-right" href="{{ pathto('conf/' + shortcode + '/' + year_str + '/schedule') }}">See the Full Schedule</a>
       </div>
 
-      {% endif %}
+      {# {% endif %} #}
```
Comment (plaindocs) [inline docs/_templates/2024/index.html]: New syntax?

**Review by plaindocs — COMMENTED** [review]
Few suggestions and questions

File: docs/_data/portland-2024-config.yaml [hunk 5]
```diff
@@ -275,3 +275,4 @@ flaghasboat: False
 flaghaswritingday: True
 flaghasjobfair: True
 flaghasbadgeflair: True
+flaghaslightningtalks: True
```
Comment (plaindocs) [inline docs/_data/portland-2024-config.yaml:277]: I don't think this exists?

Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml:277]: Yea, need to add it. Katie suggested adding all the events to the front page, which I think is a :100: idea, and done here.

File: docs/_templates/2024/index.html (hunk 4 above)
Comment (ericholscher) [reply docs/_templates/2024/index.html]: Comment :)

**Comment by ericholscher** [comment]
@plaindocs Updated this to not ship the full menu, so we can port ship the front page changes now.

File: docs/_data/portland-2024-config.yaml (hunk 2 above)
Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml]: This is copy from Katie, I think she wanted this stuff to change across time, which is definitely a bigger lift, but probably useful? 🤷

File: docs/_data/portland-2024-config.yaml (hunk 1 above)
Comment (ericholscher) [reply docs/_data/portland-2024-config.yaml:110]: Yea, I didn't know a great way to frame it. I was thinking something like `Speaker presentations`? Scheduled is not super valuable to include.

**Comment by mxsasha** [comment]
> Looking at this PR, can we replace the "has sponsors" flag with check for content in the sponsors file? Or not worth the trouble CC @mxsasha

Yeah I think we can definitely do this. Let's not block this PR on it though.

File: docs/_data/portland-2024-config.yaml (hunk 1 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml:110]: Programmed talks? Or even just Schedule?

File: docs/_data/portland-2024-config.yaml (hunk 2 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml]: Yeah, but unless we're systematic about it, we're gonna drop stuff.

File: docs/_data/portland-2024-config.yaml (hunk 5 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml:277]: Expanding the front page with more icons looks great, though I suspect the new additions need redesigning, but we don't need to block.

File: docs/_data/portland-2024-config.yaml (hunk 2 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml]: I think we can just leave it off, and ditch most of my edits here as well as we're adding further stuff below

**Review by plaindocs — APPROVED** [review]
Yeah go for it, no blockers from me.

File: docs/_data/portland-2024-config.yaml (hunk 2 above)
Comment (plaindocs) [reply docs/_data/portland-2024-config.yaml]: Actually, we could just wrap this in the conditional, although it's some pretty hairy nesting of syntax... :-D

**Pull request merged**